src/nextjournal/clerk/builder.clj

```diff
@@ -131,18 +131,15 @@
   (->> (cond paths (if (sequential? paths)
                      paths
                      (throw (ex-info "`:paths` must be sequential" {:paths paths})))
-             paths-fn (if (qualified-symbol? paths-fn)
-                        (try
-                          (if-let [resolved-var  (requiring-resolve paths-fn)]
-                            (let [resolved-paths (cond-> @resolved-var
+             paths-fn (if paths-fn
+                        (do (when-not (qualified-symbol? paths-fn)
+                              (throw (ex-info "`:path-fn` must be a qualified symbol pointing at an existing var." {:paths-fn paths-fn})))
+                            (let [resolved-var (resolve-var paths-fn)
```

Comment:
resolve-var needs to be defined before

Reply:
thanks, shouldn't push without running `clj -X:test`